skfda/exploratory/stats/_functional_transformers.py

```diff
@@ -342,3 +343,143 @@ def number_up_crossings(
         points_greater & points_smaller_rotated,
         axis=2,
     ).T
+
+
+def unconditional_central_moments(
+    data: Union[FDataBasis, FDataGrid],
+    n: int,
+) -> NDArrayFloat:
+    """
+    Calculate the unconditional central moments of a dataset.
+
+        Args:
+            data: FDataGrid or FDataBasis where we want to calculate
+            a particular central moment.
+            n: order of the moment.
+
+        Returns:
+            ndarray of shape (n_dimensions, n_samples) with the values of the
+            specified moment.
+
+    Example:
+
+    We will calculate the first unconditional central moment of the Canadian
+    Weather data set. In order to simplify the example, we will use only the
+    first five samples.
+    First we proceed to import the data set.
+    >>> from skfda.datasets import fetch_weather
+    >>> X = fetch_weather(return_X_y=True)[0]
+
+    Then we call the function with the samples that we want to consider and the
+    specified moment order.
+    >>> import numpy as np
+    >>> from skfda.exploratory.stats import unconditional_central_moments
+    >>> np.around(unconditional_central_moments(X[:5], 1), decimals=2)
+        array([[ 0.02, -0.02],
+               [ 0.03, -0.02],
+               [ 0.03, -0.01],
+               [ 0.02, -0.  ],
+               [ 0.03,  0.01]])
+    """
+    mean = np.mean(data.data_matrix, axis=1)
```

Comment:
This don't work for `FDataBasis`.

Reply:
Also, it should be the mean in the integral sense, IMHO, to consider the position of the grid points too.